### ISSUE TITLE
Static inner class ApiParamWrapper not loaded with Jersey 2.21

### ISSUE THREAD

**Issue by mihaileb** [swagger-api/swagger-core#2348]
I have this maven configuration :

		<dependency>
			<groupId>org.glassfish.jersey.core</groupId>
			<artifactId>jersey-common</artifactId>
			<version>2.23</version>
		</dependency>
		<dependency>
			<groupId>org.glassfish.jersey.media</groupId>
			<artifactId>jersey-media-json-jackson</artifactId>
			<version>2.21</version>
		</dependency>
		<dependency>
			<groupId>org.glassfish.jersey.containers</groupId>
			<artifactId>jersey-container-servlet</artifactId>
			<version>2.21</version>
		</dependency>
		<dependency>
			<groupId>org.glassfish.jersey.core</groupId>
			<artifactId>jersey-client</artifactId>
			<version>2.9</version>
		</dependency>

		

		<dependency>
			<groupId>io.swagger</groupId>
			<artifactId>swagger-jaxrs</artifactId>
			<version>1.5.9</version>
			<exclusions>
				<exclusion>
					<groupId>javax.ws.rs</groupId>
					<artifactId>jsr311-api</artifactId>
				</exclusion>
			</exclusions>
		</dependency>
		
		<dependency>
			<groupId>com.google.guava</groupId>
			<artifactId>guava</artifactId>
			<version>19.0</version>
		</dependency>

Basically, applying annotations worked using this configuration, but after some changes to the model  I receive this error :
org.glassfish.jersey.server.ContainerException: java.lang.NoClassDefFoundError: io/swagger/util/ParameterProcessor$ApiParamWrapper
 I have tried also to remove/update guava but the same exception is thrown.
Does anybody had this issue before?

**Comment by MichakrawSB** [comment]
Closing this issue as part of our https://github.com/swagger-api/swagger-core/issues/5074.
This issue is outdated and/or inactive for a long time, and we’re not able to revisit it meaningfully.

If this problem is still relevant, please open a new issue and reference this one for context.
Thank you for your understanding!
